docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
       - AUTH_SERVER_ADDRESS=http://moonshot-dev.mitre.org:8090/auth/realms/ehr/protocol/openid-connect/
       - SERVER_TITLE=MedMorph EHR
       - ADMIN_TOKEN=admin
+      - REQUIRE_AUTH=false
```

Comment:
Following Dylan's comment on https://github.com/mcode/medmorph-backend/pull/45 these should also be set to `true`. I know its the same thing as not having them at all but at least if its explicit it's easy to switch them to false